tesseract/tesseract_geometry/CMakeLists.txt

```diff
@@ -5,6 +5,9 @@ find_package(Eigen3 REQUIRED)
 find_package(console_bridge REQUIRED)
 find_package(tesseract_common REQUIRED)
 
+find_package(octomap REQUIRED)
+include_directories(${OCTOMAP_INCLUDE_DIRS})
```

Comment:
Please move this to the target that requires it.

Reply:
Done